app/pdfs/distribution_pdf.rb

```diff
@@ -178,20 +177,30 @@ def request_data
     end
 
     data += line_items.map do |c|
-      request_item = request_items.find { |i| i.item.id == c.item_id }
+      request_item = request_items.find { |i| i.item&.id == c.item_id }
+      request_qty = if Flipper.enabled?(:enable_packs) && request_item&.unit
+        "#{request_item.quantity} #{request_item.unit.pluralize(c.quantity)}"
+      else
+        request_item&.quantity || ""
+      end
       [c.item.name,
-        request_item&.quantity || "",
+        request_qty,
         c.quantity,
         dollar_value(c.item.value_in_cents),
         dollar_value(c.value_per_line_item),
         c.package_count]
     end
 
-    data += requested_not_received.sort_by(&:name).map do |c|
-      [c.item.name,
-        c.quantity,
+    data += requested_not_received.sort_by(&:name).map do |request_item|
+      request_qty = if Flipper.enabled?(:enable_packs) && request_item.unit
+        "#{request_item.quantity} #{request_item.unit.pluralize(request_item.quantity)}"
+      else
+        request_item.quantity || ""
+      end
```

Comment:
Can we extract this into a method since it's doing the same thing twice?

Reply:
Done